app/services/sources/FacebookSource.java

```diff
@@ -6,11 +6,12 @@
  * @author Reid Oliveira, Sammie Jiang
  */
 public class FacebookSource extends AbstractRestfulSource {
+	public static final String FACEBOOK = "facebook";
 	public static final String URL = "graph.facebook.com";
```

Comment:
With my curl trials either works and the API always references it without the https://